charts/bulk-scan-processor/requirements.yaml

```diff
@@ -1,6 +1,6 @@
 dependencies:
   - name: java
-    version: ~2.15.0
+    version: ~3.1.0
```

Comment:
that's a major upgrade. perhaps need to have a look at the diff and apply changes accordingly. e.g. - there's no need for readiness flag in values.yaml. requirement.yaml is deprecated. stuff like that. are we sure we are ready to jump on v3 here?

Reply:
i know it's in itself (requirements :D) but still - worth looking over rather than bumping to v3 blindly